include/CppUTestExt/MockFailure.h

```diff
@@ -48,6 +48,7 @@ class MockFailureReporter
     virtual UtestShell* getTestToFail();
 
     virtual void crashOnFailure() {crashOnFailure_ = true; }
+    virtual void clear() {crashOnFailure_ = false; }
```

Comment:
Okay. Because when using it, we might not know which test case is concerned?

How about turning it off just before we crash? In a real run, we have crashed anyway. In the test, we could be sure of no side effects.